phpBB/phpbb/user.php

```diff
@@ -301,6 +301,14 @@ function setup($lang_set = false, $style_id = false)
 		// After calling it we continue script execution...
 		phpbb_user_session_handler();
 
+		/**
+		* Execute code after that sessions are initialized
```

Comment:
@marc1706: If the event is moved at the end of the function, it will not executed if the board is disabled, so I think it's on the good location.